var/spack/repos/builtin.mock/packages/module-path-separator/package.py

```diff
@@ -13,9 +13,9 @@ class ModulePathSeparator(Package):
     version(1.0, "0123456789abcdef0123456789abcdef")
 
     def setup_run_environment(self, env):
-        env.append_path("COLON", "foo")
-        env.prepend_path("COLON", "foo")
-        env.remove_path("COLON", "foo")
```

Comment:
Why was it necessary to change this?

Reply:
The default behavior for environment path modifications on windows is to use a semicolon as the separator for lists. Without this change there's a mismatch between the label the separator used. It does shift the test away from using the default behavior however the default behavior is unit-tested [here](https://github.com/spack/spack/blob/221cf2e3ebb2675728b7f548946a0e28864c1430/lib/spack/spack/test/environment_modifications.py#L182)